lvgl_helpers.c

```diff
@@ -19,9 +19,17 @@
 #include "driver/i2c.h"
 
 #ifdef LV_LVGL_H_INCLUDE_SIMPLE
-#include "src/lv_core/lv_refr.h"
+#if (LVGL_VERSION_MAJOR < 8)
```

Comment:
For the sake of simplicity, I think we can include `lvgl.h` directly.

Reply:
Thanks, I will update it.

Reply:
Like this?
```diff
diff --git a/lvgl_helpers.c b/lvgl_helpers.c
index 56c9183..a215c43 100644
--- a/lvgl_helpers.c
+++ b/lvgl_helpers.c
@@ -19,17 +19,9 @@
 #include "driver/i2c.h"

 #ifdef LV_LVGL_H_INCLUDE_SIMPLE
-#if (LVGL_VERSION_MAJOR < 8)
-    #include "src/lv_core/lv_refr.h"
-#else
-    #include "src/core/lv_refr.h"
-#endif
+#include "lvgl.h"
 #else
-#if (LVGL_VERSION_MAJOR < 8)
-    #include "lvgl/src/lv_core/lv_refr.h"
-#else
-    #include "lvgl/src/core/lv_refr.h"
-#endif
+#include "lvgl/lvgl.h"
 #endif
```